src/kraken/std/python/tasks/info_task.py

```diff
@@ -0,0 +1,83 @@
+import re
+import subprocess
+from pathlib import Path
+from typing import List
+
+from kraken.core.api import Project, Property, Task, TaskStatus
+
+from ..buildsystem import PythonBuildSystem
+
+
+def get_python_path() -> str:
+    """Returns Python path"""
+    path = run_command(["which", "python"])
+    if "command not found" in path:  # Windows
+        path = run_command(["where", "python"])
+    return path
+
+
+def get_python_version() -> str:
+    """Returns Python version"""
+    return run_command(["python", "--version"])
+
+
+def check_python_version(version: str) -> None:
+    """Checks if Python is in the correct version"""
+    version_index = re.search(r"(\d+\.)?(\d+\.)?(\d+)$", version)
+    if version_index:
+        version = version[version_index.span()[0] :]
+        [major, minor, _] = version.split(".")
+        if int(major) != 3 or int(minor) != 10:
+            print(f"[WARNING]: current version is {version}, it should be 3.10, please check your python settings")
+
+
+def run_command(command: List[str]) -> str:
+    """Run a subprocess command and decodes the output"""
+    return subprocess.run(command, stdout=subprocess.PIPE, shell=False).stdout.decode("utf-8")
+
+
+class InfoTask(Task):
+    build_system: Property[PythonBuildSystem]
+
+    def get_description(self) -> str:
+        return """Show several python information such as:\n
+        - Python version
+        - Python path
+        - Virtual environment path
+        """
+
+    def execute(self) -> TaskStatus:
+        """
+        Shows Python info such as:
+            - Python version
+            - Python path
+            - Virtual environment path
+        """
+        version = get_python_version()
+        python_path = get_python_path()
+        virtual_env_path = self.get_virtual_env_path()
+        check_python_version(version)
+        print(" ---------- Python info ----------")
+        print(f"Python version: {version}")
+        print(f"Python path: {python_path}")
+        print(f"Virtual environment path: {virtual_env_path}")
+        print(" ---------------------------------")
+
+        return TaskStatus.succeeded()
+
+    def get_virtual_env_path(self) -> Path:
+        """
+        Returns the current virtual environment path
+        """
+        managed_env = self.build_system.get().get_managed_environment()
+        path = managed_env.get_path()
+        return path
```

Comment:
let's just inline everything

```suggestion
        return self.build_system.get().get_managed_environment().get_path()
```